vars/getBuildManifestInfo.groovy

```diff
@@ -0,0 +1,16 @@
+Map call(Map args = [:]) {
+    def lib = library(identifier: "jenkins@20211123", retriever: legacySCM(scm))
+
+    String buildManifestFileName = 'manifest.yml'
+    echo "Setting BUILD_MANIFEST=${buildManifestFileName}"
+    env.BUILD_MANIFEST = buildManifestFileName
+    
+    sh "wget ${BUILD_MANIFEST_URL} -O ${buildManifestFileName}"
+    def buildManifest = lib.jenkins.BuildManifest.new(readYaml(file: "${buildManifestFileName}"))
+    String architecture = buildManifest.getArtifactArchitecture()
+    String buildId = buildManifest.getArtifactBuildId()
+    echo "Setting ARCHITECTURE=${architecture}"
+    env.ARCHITECTURE = architecture
+    echo "Setting BUILD_ID=${buildId}"
+    env.BUILD_ID = buildId
```

Comment:
I would prefer that we return these as an output of this function call if possible

Reply:
Fixed in https://github.com/opensearch-project/opensearch-build/commit/e13948505ded132571d25c0f805ae499f9dfaf46